### DISCUSSION TITLE
Welcome to OSPO Discussions!

### DISCUSSION THREAD

**Discussion by ahpook** [community/OSPO#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Hermes00** [comment]
Hello, Happy Hacking to everyone! I'm trying to get assistance for a team from Space Apps Rosario, who could help? Thank you!                         "we'd like to have access to some data about average and high values of speed, temperature, and Bx, By and Bz movements of the IMF. It'd be very helpful if you could also explain us how to read and analise that information, thank you".
